frontend/src/tabs/zoom.pug

```diff
@@ -46,15 +47,15 @@
 
   .sorting.mui-form--inline
     .mui-select
-      select.medium-font(v-model="commitsSortType")
+      select(v-model="commitsSortType")
         option(value="time") Time
         option(value="lineOfCode") LoC
-      label.medium-font sort by
+      label sort by
     .mui-select
-      select.medium-font(v-model="toReverseSortedCommits")
+      select(v-model="toReverseSortedCommits")
         option(v-bind:value='true') Descending
         option(v-bind:value='false') Ascending
-      label.medium-font order
+      label order
```

Comment:
this

Reply:
This is overwritten in the original css. Therefore, removed it.